cmd/pgbackrest/main.go

```diff
@@ -240,3 +168,180 @@ func NewForConfig(config *rest.Config) (*KubeAPI, error) {
 
 	return &api, err
 }
+
+// getEnvRequired attempts to get an environmental variable that is required
+// by this program. If this cannot happen, we fatally exit
+func getEnvRequired(envVar string) string {
+	val := strings.TrimSpace(os.Getenv(envVar))
+
+	if val == "" {
+		log.Fatalf("required environmental variable %q not set, exiting.", envVar)
+	}
+
+	log.Debugf("%s set to: %s", envVar, val)
+
+	return val
+}
+
+// loadConfiguration loads configuration from the environment as needed to run a pgBackRest
+// command
+func loadConfiguration(ctx context.Context, kubeapi *KubeAPI) config {
+	cfg := config{}
+
+	cfg.namespace = getEnvRequired("NAMESPACE")
+	cfg.command = getEnvRequired("COMMAND")
+
+	cfg.container = os.Getenv("CONTAINER")
+	if cfg.container == "" {
+		cfg.container = containerNameDefault
+	}
+	log.Debugf("CONTAINER set to: %s", cfg.container)
+
+	cfg.commandOpts = os.Getenv("COMMAND_OPTS")
+	log.Debugf("COMMAND_OPTS set to: %s", cfg.commandOpts)
+
+	cfg.selector, _ = os.LookupEnv("SELECTOR")
```

Comment:
⛏️ This is the only use of LookupEnv in the file, and iiuc using it this way is the same as Getenv.
```suggestion
	cfg.selector = os.Getenv("SELECTOR")
```

Reply:
Switched to `Getenv()`.